catroid/src/main/java/org/catrobat/catroid/CatroidApplication.java

```diff
@@ -82,6 +83,48 @@ public static synchronized boolean loadNativeLibs() {
 		return parrotLibrariesLoaded;
 	}
 
+	public static synchronized boolean loadSDKLib() {
+		if (parrotJSLibrariesLoaded) {
+			return true;
+		}
+
+		try {
+			System.loadLibrary("curl");
```

Comment:
that's a bunch of native libraries. do they all come with the com.parrot:arsdk ?